index.html

```diff
@@ -1434,7 +1434,7 @@ <h2>Resolver Architectures</h2>
 			<p>Also describe mitigations against potential downsides of <a>remote bindings</a>, e.g.:</p>
 			<ul>
 				<li>A <a>DID resolver's</a> <a>remote binding</a> can use a trusted channel such as VPN or TLS with mutual authentication.</li>
-				<li>A <a>DID resolver</a> can add a <a href="https://w3c.github.io/ld-proofs/">proof</a>
+				<li>A <a>DID resolver</a> can add a <a href="https://www.w3.org/TR/vc-data-integrity/">data integrity proof</a>
```

Comment:
Should this be an a tag? Isn't there a ReSpec way to reference this specification so that it appears at the bottom of the document as a reference

Reply:
@wip-abramson Just added a commit that addresses this.